RELEASE-howto.md

```diff
@@ -0,0 +1,121 @@
+# Howto release PyWPS
+
+This document gives you, as PyWPS release master complete tutorial of how to get
+PyWPS release rolled up and deployed to target server, create packages etc.
+
+## PyWPS versioning
+
+PyWPS uses [Debian version naming system](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version). Every policy should be checked against it.
+
+PyWPS uses 3 numbers release description: MAJOR.MINOR.MAINTENANCE. Currently,
+the MAJOR version is `4`. Within MAJOR releases, we should aim, not to break
```

Comment:
Suggest we remove the `Currently, the MAJOR version is\`4`` sentence given this will change over time.